data/sql/updates/pending_db_world/rev_1735053928227867164.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `smart_scripts`
```

Comment:
Delete before Insert to avoid any potential conflict

Reply:
Fixed. I also added all PK for UPDATE.